site/content/api/Telemetry-API.md

```diff
@@ -60,11 +60,13 @@ The following sequence diagram illustrates the flow of messages involved in the
 
 The following table provides an overview of the properties a client needs to set on an *Upload Telemetry Data* message.
 
-| Name           | Mandatory | Location                 | Type      | Description |
-| :------------- | :-------: | :----------------------- | :-------- | :---------- |
-| *content-type* | yes       | *properties*             | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
-| *device_id*    | yes       | *application-properties* | *string*  | MUST contain the ID of the device the data in the payload has been reported by. |
-| *reg_assertion*| yes       | *application-properties* | *string*  | A [JSON Web Token](https://jwt.io/introduction/) issued by the [Device Registration service]({{< relref "api/Device-Registration-API.md#assert-device-registration" >}}) asserting the device's registration status. |
+| Name            | Mandatory | Location                 | Type      | Description |
+| :-------------- | :-------: | :----------------------- | :-------- | :---------- |
+| *content-type*  | yes       | *properties*             | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
+| *creation-time* | yes       | *properties*             | *timestamp* | MUST be set to the current number of milliseconds since the unix epoch (1970-01-01T00:00:00Z). |
```

Comment:
since this is the standard property defined by AMQP 1.0 why not simply use the definition from the spec (which also notes that the point in time it refers to is supposed to be UTC).